lite-rpc/src/lib.rs

```diff
@@ -28,7 +28,7 @@ pub const DEFAULT_FANOUT_SIZE: u64 = 10;
 #[from_env]
 pub const MAX_RETRIES: usize = 40;
 
-pub const DEFAULT_RETRY_TIMEOUT: u64 = 2;
+pub const DEFAULT_RETRY_TIMEOUT: u64 = 1;
```

Comment:
can this be _Duration_ typed or have at least a suffix and/or comment if it's seconds/milliseconds/etc.

Reply:
Duration is used in CLI as a default and Duration is incompatible with Clap

Reply:
k. got it, that's sad